terraform/modules/tf_rules_engine/main.tf

```diff
@@ -0,0 +1,19 @@
+// SNS Topic Subscription: Subscribe the Classifier to configured SNS topics
```

Comment:
Looks these code is for Classifier. Can you remove it?

Reply:
yes!! good catch :)